insights/parsers/tests/test_teamdctl_state_dump.py

```diff
@@ -56,3 +81,32 @@ def test_teamdctl_state_dump():
     assert result['runner']['active_port'] == 'eno1'
     assert result['setup']['runner_name'] == 'activebackup'
     assert result.runner_type == 'activebackup'
+    assert result.team_ifname == 'team0'
+
+
+def test_teamdctl_state_dump_none():
+    result = TeamdctlStateDump(context_wrap(TEAMDCTL_STATE_DUMP_INFO_NONE))
+
+    assert result.data == {
+        'runner': {
+        },
+        'setup': {
+            'daemonized': False,
+            'zmq_enabled': False,
+            'kernel_team_mode_name': 'activebackup',
+            'pid': 4464,
+            'dbus_enabled': True,
+            'debug_level': 0,
+            'pid_file': '/var/run/teamd/team0.pid',
+            'runner_name': 'activebackup'
+        },
+        'team_device': {
+            'ifinfo': {
+                'ifindex': 5,
+                'dev_addr': '2c:59:e5:47:a9:04',
+                'dev_addr_len': 6}
+        }
+    }
+    assert result['setup']['runner_name'] == 'activebackup'
+    assert result.runner_type == 'activebackup'
+    assert result.team_ifname is None
```

Comment:
I have given an example of team_ifname where team_ifname returns a valid name in line 84:
assert result.team_ifname == 'team0'